frontend/src/components/Logs/ListLogView/index.tsx

```diff
@@ -145,82 +146,59 @@ function ListLogView({
 		[flattenLogData.timestamp],
 	);
 
+	const logType = logData?.attributes_string?.log_level || LogType.INFO;
+
+	const handleMouseEnter = (): void => {
+		setHasActionButtons(true);
+	};
+
+	const handleMouseLeave = (): void => {
+		setHasActionButtons(false);
+	};
+
 	return (
-		<Container $isActiveLog={isHighlighted}>
-			<div>
-				<LogContainer>
-					<>
-						<LogGeneralField fieldKey="log" fieldValue={flattenLogData.body} />
+		<Container
+			$isActiveLog={isHighlighted}
```

Comment:
$isActiveLog ?

Reply:
this is for styled components styling , existing code